vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -158,6 +160,11 @@ def build(self):
         for page in self.pages:
             page.build()  # TODO: ideally remove, but necessary to register slider callbacks
 
+        # Define callbacks when the dashboard is built but not every time the page is changed.
```

Comment:
Previously these callbacks were defined by code that lived inside `ActionLoop._create_app_callbacks()`. Now it's done explicitly here but effectively still defined in the same place.